requirements.txt

```diff
@@ -1,2 +1,4 @@
+pillow
 pyramid
-pyramid_jinja2
\ No newline at end of file
+pyramid_jinja2
+sqlalchemy
```

Comment:
It seems we can remove this and also the file called `models.py` -- it's not used anywhere.

Reply:
Sounds good!

Reply:
Done.